cluster/addons/addon-manager/Makefile

```diff
@@ -30,6 +30,10 @@ endif
 ifeq ($(ARCH),ppc64le)
 	BASEIMAGE?=ppc64le/debian
 endif
+ifeq ($(ARCH),s390x)
+	BASEIMAGE?=s390x/python:2.7-slim
+	QEMUARCH=s390x
```

Comment:
Actually I had to mention "s390x/debian" only but due to some wrong commit merges it got changed.